arch/arm/src/imxrt/imxrt_serial.c

```diff
@@ -60,6 +60,24 @@
 
 #ifdef USE_SERIALDRIVER
 
+#if defined(SERIAL_HAVE_TXDMA)
```

Comment:
the TCD pool is shared across all devices.  So this is only a Band-Aid. For the minimal case

Reply:
totally agree. I just found myself using a single lpuart device with DMA for the first time and it wasn't obvious to me why dma wasn't working properly (no transfers were occurring) until I stumbled upon the fact that you need to set `CONFIG_IMXRT_EDMA_NTCD` appropriately. I wasn't sure of a better way to communicate this to the user and was hoping these compile-time checks would help at least a few people :)